qa/tests/features/new/manual/filter-refiner/Range Filter/Filter Refiner-Range Filter.feature

```diff
@@ -0,0 +1,127 @@
+	And the range <MIN #2> <= <Attribute Name#2> <= <MAX #2> should be displayed in the right part of the screen
+	And the variants number should be equal to <Variants Number>
+	And the "View Variants" dialog can be opened
+	And the "Full List" view should be disabled
+
+Examples:
+	| Attribute Name#1 | MIN #1 | MAX #1 | Attribute Name#2 | MIN #2 | MAX #2 | Variants Number |
+	| Severity         | -1     | 0      | Proband_GQ       | 100    | 200    | 35994           |
+
+
+Scenario Outline: 04 Filter by range with invalid values (Minimum value < MIN)
+	When user clicks the <Attribute Name>
+	And enters the minimum value of range <Minimum Value>
+	And enters the maximum value of range <Maximum Value>
+	Then invalid value should be changed to valid value
+	And "+ Add Attribute" button should be enabled
+
+Examples:
+	| Filter name                  | Minimum value | Maximum value |
+	| Num_Samples                  | -1            | 3             |
+	| Num_Genes                    | -1            | 23            |
+	| Start_Pos                    | 0             | 248930384     |
+	| gnomAD_PopMax_AN             | -1            | 249160        |
+	| Number_submitters            | -1            | 30            |
+	| Min_GQ                       | -2            | 840           |
+	| Number_of_clinvar_submitters | -1            | 30            |
+	| Severity                     | -2            | 3             |
+
+Scenario Outline: 05 Filter by range with invalid values (Maximum value > MAX)
+	When user clicks the <Attribute Name>
+	And enters the minimum value of range <Minimum Value>
+	And enters the maximum value of range <Maximum Value>
+	Then invalid value should be changed to valid value
+	And "+ Add Attribute" button should be enabled
+
+Examples:
+	| Attribute Name               | Minimum Value | Maximum Value |
+	| Num_Samples                  | 0             | 4             |
+	| Num_Genes                    | 0             | 24            |
+	| Start_Pos                    | 25165         | 248930385     |
+	| gnomAD_PopMax_AN             | 0             | 249161        |
+	| Number_submitters            | 0             | 31            |
+	| Min_GQ                       | -1            | 841           |
+	| Number_of_clinvar_submitters | 0             | 31            |
+	| Severity                     | -1            | 4             |
+
+Scenario Outline: 06 Clear button
+	When user clicks the <Attribute Name>
+	And enters the minimum value of range <Minimum Value>
+	And changes "<" sign with "≤" next to the minimum value
+	And enters the maximum value of range <Maximum Value>
+	And clicks the "+ Add Attribute" button to apply the filter
+	And the filter is applied
+	And the user clicks the "Clear" button
+	Then the filter should be cleared
+
+Examples:
+	| Attribute Name | Minimum Value | Maximum Value | Total variants |
+	| Num_Samples    | 1             | 2             | 3281689        |
```

Comment:
Add please scenario that when user inputs values of an interval it is displayed on the diagram